### PR TITLE
refactor: remove slots per epoch

### PR DESCRIPTION
**Pull request by barnabasbusa**

### CODE REVIEW

File: src/package_io/parse_input.star [hunk 1]
```diff
@@ -296,7 +292,7 @@ def parse_network_params(input_args):
     ):
         fail("electra can only happen with capella genesis not bellatrix")
 
-    required_num_validators = 2 * result["network_params"]["slots_per_epoch"]
```
Comment (h4ck3rk3y) [inline src/package_io/parse_input.star:299]: why are you hard coding this?

File: static_files/cl-forkmon-config/config.toml.tmpl [hunk 2]
```diff
@@ -4,7 +4,6 @@ etherscan_api_key = "some-etherscan-api-key"
 
 [consensus_chain]
 seconds_per_slot = {{ .SecondsPerSlot }}
-slots_per_epoch = {{ .SlotsPerEpoch }}
```
Comment (h4ck3rk3y) [inline static_files/cl-forkmon-config/config.toml.tmpl:7]: why are we removing this?

File: src/package_io/parse_input.star (hunk 1 above)
Comment (barnabasbusa) [reply src/package_io/parse_input.star:299]: fixed

Comment (h4ck3rk3y) [reply src/package_io/parse_input.star:298]: oh my multi tasking is failing, i didn't see that this depends on `slots_per_epoch`

File: src/package_io/parse_input.star [hunk 3]
```diff
@@ -295,15 +294,14 @@ def parse_network_params(input_args):
...
     actual_num_validators = (
         total_participant_count
         * result["network_params"]["num_validator_keys_per_node"]
     )
-    if required_num_validators > actual_num_validators:
+    if MIN_VALIDATORS > actual_num_validators:
         fail(
-            "required_num_validators - {0} is greater than actual_num_validators - {1}".format(
-                required_num_validators, actual_num_validators
+            "MIN_VALIDATORS - {0} is greater than actual_num_validators - {1}".format(
```
Comment (h4ck3rk3y) [inline src/package_io/parse_input.star:303]: can we do something like

"We require at least {0} validators but got {1}", MIN_VALIDATORS, actual_num_validators"

**Review by h4ck3rk3y — APPROVED** [review]

**Pull request merged**